man/corosync.conf.5

```diff
@@ -420,16 +431,24 @@ the library's documentation for more detailed information.
 .TP
 hold
 This timeout specifies in milliseconds how long the token should be held by
-the representative when the protocol is under low utilization.   It is not
-recommended to alter this value without guidance from the corosync community.
+the representative when the protocol is under low utilization. If not specified in
+config file, it will be automatically calculated if token_retransmit (explained in
```

Comment:
Very similar story. No matter if `token_retransmit` is modified or not. Hold is always 'totem_config->token_retransmit_timeout * 0.8 - (1000/HZ)' if not specified explicitly.

Reply:
Get your point. Same with previous, what I want to explain is relations amount those three, in order to keep users from specified wrong cause error occur.